Common/Core/RecoDecay.h

```diff
@@ -728,7 +714,7 @@ class RecoDecay
       if (!arrDaughters[iProng].has_mcParticle()) {
         return -1;
       }
-      auto particleI = arrDaughters[iProng].template mcParticle_as<aod::McParticles_000>(); // ith daughter particle
+      auto particleI = arrDaughters[iProng].template mcParticle_as<aod::McParticles>(); // ith daughter particle
```

Comment:
This could also become `auto particleI = arrDaughters[iProng].mcParticle();`

Reply:
Done!